RecoTracker/IterativeTracking/python/ElectronSeeds_cff.py

```diff
@@ -16,8 +16,8 @@
     oldClusterRemovalInfo = cms.InputTag("initialStepSeedClusterMask")
 )
 eras.trackingPhase1PU70.toModify(pixelPairStepSeedClusterMask, oldClusterRemovalInfo = "highPtTripletStepSeedClusterMask")
-# This is a pure guess to use lowPtTripletStep for phase1 here instead of the pixelPair in Run2 configuration
-lowPtTripletStepSeedClusterMask = seedClusterRemover.clone(
+# This is a pure guess to use detachedTripletStep for phase1 here instead of the pixelPair in Run2 configuration
+detachedTripletStepSeedClusterMask = seedClusterRemover.clone(
```

Comment:
I'm not really sure about the necessity of the changes in this file (last commit, 3b96736) as they seem to have no effect in gsfElectrons (within 1000 ttbar+PU events), but this way the cluster masking is more consistent with run2 tracking (using last pixel-only iteration).